backend/src/packages/articles/article.controller.ts

```diff
@@ -69,6 +69,19 @@ class ArticleController extends Controller {
 
     this.articleService = articleService;
 
+    this.addRoute({
+      path: ArticlesApiPath.ROOT,
+      method: 'GET',
+      handler: () => this.findAll(),
+    });
+
+    this.addRoute({
+      path: ArticlesApiPath.$ID,
```

Comment:
If this route is for getting all users articles and you're passing user's ID - then it should not work this way
Better call route something like `/own` and just get userId from authorized user on backend

Reply:
/articles/:id -- should return specific article by it's ID